openjdk/mmtkBarrierSet.hpp

```diff
@@ -37,6 +37,9 @@
 #include "mmtkBarrierSetAssembler_x86.hpp"
 
 #define MMTK_ENABLE_ALLOCATION_FASTPATH true
+#define MMTK_ENABLE_GLOBAL_ALLOC_BIT false
+// hard coded for now
+const intptr_t ALLOC_BIT_BASE_ADDRESS = 0x0000060000000000;
```

Comment:
If you want to get the address, you can do something like this: https://github.com/mmtk/mmtk-openjdk/blob/648bb27f4bb5c84a83f74fda1d699514548cf663/openjdk/mmtk.h#L15

Reply:
See this PR: https://github.com/mmtk/mmtk-openjdk/pull/86